after/plugin/dracula.vim

```diff
@@ -97,6 +97,11 @@ if exists('g:loaded_nvim_treesitter')
   " HTML and JSX tag attributes. By default, this group is linked to TSProperty,
   " which in turn links to Identifer (white).
   hi! link TSTagAttribute DraculaGreenItalic
+	" CSS
+	hi! link TSPunctDelimiter DraculaPink
+	hi! link TSType DraculaPink
+	hi! link TSProperty DraculaCyan
+	hi! link TSError DraculaOrange
```

Comment:
I'm not sure this is going to be the proper way to go about this. `TSType`, for example, is the token used for all `type`s, in ever language. That for the most part should be `DraculaCyan`..

The way we need to go about this is be specific to these changes _just_ for css languages. There's a way to target those languages specifically, but you'll have to look into the docs for the tree sitter plugin to find that out because I can't recall off the top of my head how to do that.

Reply:
Seconded

Reply:
yea, you might make some queries for that.

For example, create `queries/css/highlights.scm` in the plugin's directory like this:
```scm
; inherits: css

[
 "#"
 ","
 "."
 ":"
 "::"
 ";"
 ] @keyword

(class_name) @type
(property_name) @type
(nesting_selector) @keyword
```

This will partially do the thing you did w/out linking any highlight groups 👽
![image](https://user-images.githubusercontent.com/13046595/197648878-e67e6753-3b49-4847-8759-231940e176d6.png)

p.s. you might find helpful `:TSPlaygroundToggle`, `:TSNodeUnderCursor` and `TSHighlightCapturesUnderCursor` commands of the `nvim-treesitter/playground` addon on your way.